en/console-and-shells/completion-shell.rst

```diff
@@ -94,20 +106,50 @@ The following bash example comes from the original author::
             opts=$(${cake} Completion subcommands $prev)
             COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
             if [[ $COMPREPLY = "" ]] ; then
-                COMPREPLY=( $(compgen -df -- ${cur}) )
+                _filedir
```

Comment:
@cake17 Thanks for the feedback ! I'll update the PR to include those informations for users using Mac OSX